plugins/github-actions/src/components/useWorkflowRuns.ts

```diff
@@ -43,53 +43,48 @@ export function useWorkflowRuns({
   const { loading, value: runs, retry, error } = useAsyncRetry<
     WorkflowRun[]
   >(async () => {
-    return (
-      api
-        // GitHub API pagination count starts from 1
-        .listWorkflowRuns({
-          hostname,
-          owner,
-          repo,
-          pageSize,
-          page: page + 1,
-          branch,
-        })
-        .then((workflowRunsData): WorkflowRun[] => {
-          setTotal(workflowRunsData.total_count);
-          // Transformation here
-          return workflowRunsData.workflow_runs.map(run => ({
-            workflowName: run.name,
-            message: run.head_commit.message,
-            id: `${run.id}`,
-            onReRunClick: async () => {
-              try {
-                await api.reRunWorkflow({
-                  hostname,
-                  owner,
-                  repo,
-                  runId: run.id,
-                });
-              } catch (e) {
-                errorApi.post(e);
-              }
-            },
-            source: {
-              branchName: run.head_branch,
-              commit: {
-                hash: run.head_commit.id,
-                url: run.head_repository.branches_url.replace(
-                  '{/branch}',
-                  run.head_branch,
-                ),
-              },
-            },
-            status: run.status,
-            conclusion: run.conclusion,
-            url: run.url,
-            githubUrl: run.html_url,
-          }));
-        })
-    );
+    // GitHub API pagination count starts from 1
+    const workflowRunsData = await api.listWorkflowRuns({
```

Comment:
Sorry for the noise, reformatted to use `await` instead of `.then` for minor readability improvement.